cli/cmd/install/install.go

```diff
@@ -203,6 +207,36 @@ func (i *Install) getConfigMap(ctx *clicontext.CLIContext, ignoreCluster bool) (
 		}
 	}
 
+	// configure letsencrypts email
+	if !i.NoEmail && !i.Check && !slice.ContainsString(i.DisableFeatures, "letsencrypt") {
+		if i.Email != "" {
+			cfg.LetsEncrypt.Email = i.Email
+		} else {
+			msg := "Please provide your Let's Encrypt email\n\n"
+			options := []string{
+				"[1]: Provide an Email address.\n\tThis is used to send you important notifications, certificate expiration warnings, and for lost account recovery.\n\tIt will never be shared with Rancher.\n",
+				"[2]: Use Let's Encrypt with no email address.\n\tThis is strongly discouraged and lack of notifications may cause you to lose access to your certificates.\n",
+				"[3]: Do not use Let's Encrypt at all.\n\tCert-manager will not be deployed and no certificates will be automatically issued for you.\n\tYou will not be able to use dashboard unless you configure custom cluster domain and wildcard certificates.\n",
+			}
+
+			num, err := questions.PromptOptions(msg, -1, options...)
+			if err != nil {
+				return cfg, nil, err
+			}
+			if num == 0 {
+				email, err := questions.Prompt("Please enter your email address:\n", "your@company.com")
+				if err != nil {
+					return cfg, nil, err
+				}
+				cfg.LetsEncrypt.Email = email
+			} else if num == 1 {
+				logrus.Warnf("No email is provided. This is strongly discouraged and lack of notifications may cause you to lose access to your certificates. Run `rio systemconfig --edit` to add it back")
```

Comment:
Are we sure adding an email well after setup has completed will actually send it to LE ?

Reply:
what do you mean?

Reply:
If I install rio with option 2, and then 4 weeks later I update the config to include my email address, will that email get sent to letsencrypt and attached to that existing cert ?

Reply:
I don't think it will take effect immediately, letsencrypts will probably have that information next time it renews certs. But we have nothing to do on our side to change this.